v2/src/log_context_property_basic_types.c

```diff
@@ -27,6 +34,27 @@
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_033: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint32_t).to_string shall succeed and return the result of snprintf. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_034: [ If any error is encountered, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint32_t).to_string shall fail and return -1. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_042: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).to_string shall call snprintf with buffer, buffer_length and format string PRId16 and pass in the values list the int16_t value pointed to be property_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_043: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).to_string shall succeed and return the result of snprintf. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_044: [ If any error is encountered, LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).to_string shall fail and return -1. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_052: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).to_string shall call snprintf with buffer, buffer_length and format string PRId16 and pass in the values list the uint16_t value pointed to be property_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_053: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).to_string shall succeed and return the result of snprintf. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_054: [ If any error is encountered, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).to_string shall fail and return -1. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_062: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).to_string shall call snprintf with buffer, buffer_length and format string PRId8 and pass in the values list the int8_t value pointed to be property_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_063: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).to_string shall succeed and return the result of snprintf. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_064: [ If any error is encountered, LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).to_string shall fail and return -1. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_072: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).to_string shall call snprintf with buffer, buffer_length and format string PRId8 and pass in the values list the uint8_t value pointed to be property_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_073: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).to_string shall succeed and return the result of snprintf. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_074: [ If any error is encountered, LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).to_string shall fail and return -1. ]*/ \
             result = snprintf(buffer, buffer_size, "%" print_format_string "", *(type_name*)property_value); \
```

Comment:
>result = snprintf [](http://example.com/codeflow?start=12&length=17)

So this is interesting, as the caller of this `int` function will have to compare the returned result to see "did it fit" (returned result is less than "n") / "did it work" (returned result is greater than 0)

So this is really interesting, looking forward to the caller to see how the information is used!